lib/system/external.toit

```diff
@@ -17,20 +35,32 @@ class Client:
 
   constructor.private_ .pid .id .on-notify_:
 
-  static open id/string --on-notify/Lambda?=null -> Client?:
+  /**
+  Opens a connection to an external process with the given $id.
```

Comment:
Maybe 'opens a client for an ...'. Not sure it helps to introduce the notion of a connection here.

Reply:
done.